icechunk-python/python/icechunk/_icechunk_python.pyi

```diff
@@ -77,32 +77,32 @@ class CachingConfig:
 
     def __init__(
         self,
-        snapshots_cache_size: int,
-        manifests_cache_size: int,
-        transactions_cache_size: int,
-        attributes_cache_size: int,
-        chunks_cache_size: int,
+        snapshots_nodes: int,
```

Comment:
I'd like the user-facing parameter names be easier to interpret (i didn't understand the units till I read the code). So how about
```
num_nodes
num_manifests
num_transaction_logs
nbytes_attributes
nbytes_chunks
```